src/Altinn.App.Api/Controllers/ResourceController.cs

```diff
@@ -250,5 +250,24 @@ public ActionResult GetRuleConfiguration(string org, string app, string id)
 
             return new FileContentResult(fileContent, MimeTypeMap.GetMimeType(".json"));
         }
+
+        /// <summary>
+        /// Get the footer layout
+        /// </summary>
+        /// <param name="org">The application owner short name</param>
+        /// <param name="app">The application name</param>
+        /// <returns>The footer layout in the form of a string.</returns>
+        [HttpGet]
+        [Route("{org}/{app}/api/v1/footer")]
+        public ActionResult GetFooterLayout(string org, string app)
+        {
+            string layout = _appResourceService.GetFooter();
```

Comment:
Sorry for missing this in the previous batch. I tend to prefer `var`
```suggestion
            string? layout = _appResourceService.GetFooter();
```